src/components/character-count/_character-count.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+@import "../error-message/error-message";
+@import "../hint/hint";
+@import "../label/label";
+
+@include govuk-exports("govuk/component/character-count") {
+  .govuk-character-count {
+    .govuk-textarea {
```

Comment:
Is this safe? We're relying on the textarea here.

I wonder if `.govuk-character-count__textarea` would be better, but I assume these overrides only make sense when this component is interacting with the other one's styles...